### ISSUE TITLE
When entering integer values in VDR OSD only "0" and "1" button work

### ISSUE THREAD

**Issue by M-Reimer** [FernetMenta/pvr.vdr.vnsi#20]
I really like your feature to control the VDR OSD from Kodi.

I only use Kodi as frontend so this is the only way for me to change settings.

If I want to edit an "Int" value in plugin settings, then for some reason only the "0" and "1" button can be used to enter something. All the other number buttons are dead for me.

Can this be improved, so that all the number buttons are communicated to VDR?

**Comment by FernetMenta** [comment]
Did you map the number buttons to the addon like the colour buttons?
https://github.com/xbmc/xbmc/blob/master/system/keymaps/remote.xml#L709

**Comment by M-Reimer** [comment]
Exactly. I'm using my remote control via LIRC. Only the "0" and "1" buttons work on my remote. All the others don't work for entering numbers.

**Comment by FernetMenta** [comment]
Do the number buttons work with other windows?

**Comment by M-Reimer** [comment]
I've tried with normal "live TV" in Kodi (entering channel numbers) and all my number buttons work there.

Maybe I should've said that I'm still on one of the Kodi 18 Betas. Was a bit busy recently and didn't have time to build the RC versions. I want to wait for the "final" now, which will then also be shipped by my distributor.

**Comment by FernetMenta** [comment]
Maybe something got broken. I'll have a look later.

**Comment by FernetMenta** [comment]
I tested with a keyboard and it worked. I went to setup - plugins - vnsi and entered 7 for timeshift buffer size.
Maybe an issue with the setup menu of the plugin? What plugin is this? Can you do the same test as I did?
